entries/p/pro.btcturk.com.json

```
@@ -0,0 +1,16 @@
+{
+  "BtcTurk Pro": {
+    "domain": "pro.btcturk.com",
+    "tfa": [
+      "sms",
+      "email",
+      "totp"
+    ],

```

A documentation page is preferred. This seems like a good one:
```suggestion
    ],
    "documentation": "https://www.btcturk.com/yardim/guvenlik",
```